addons/survey/models/survey_survey.py

```diff
@@ -37,9 +38,14 @@ def _get_default_access_token(self):
     question_and_page_ids = fields.One2many('survey.question', 'survey_id', string='Sections and Questions', copy=True)
     page_ids = fields.One2many('survey.question', string='Pages', compute="_compute_page_and_question_ids")
     question_ids = fields.One2many('survey.question', string='Questions', compute="_compute_page_and_question_ids")
-    stage_id = fields.Many2one('survey.stage', string="Stage", default=lambda self: self._get_default_stage_id(),
-                               ondelete="restrict", copy=False, group_expand='_read_group_stage_ids')
-    is_closed = fields.Boolean("Is closed", related='stage_id.closed', readonly=True)
+    state = fields.Selection(
```

Comment:
It should be required to avoid having surveys in undefined states.

Why putting it readonly ?